.github/workflows/ci.yml

```diff
@@ -526,7 +591,7 @@ jobs:
                   cat "$previous_file" >> "$diff_output"
               fi
           done
-          
+
```

Comment:
It's true it's annoying. But style changes are fine. In the future we should have (non-annoying) rules so that things like trailing spaces are removed, each of us in our own IDE.
One thing that is wrong is mixing different things in the same PR. So I'm going to create a small PR to modify these style changes, and when it's merged into develop this will no longer be in the way of this PR and will not hinder its review.

Reply:
https://github.com/cppalliance/mrdocs/pull/651

Reply:
> It's true it's annoying. But style changes are fine. 

I agree with annoying more than fine. lol

> In the future we should have (non-annoying) rules so that things like trailing spaces are removed, each of us in our own IDE.

We don't need rules for that. It's just a matter of not changing the lines, whatever they look like currently. The rule is "don't introduce unintended changes in formatting".

> One thing that is wrong is mixing different things in the same PR. So I'm going to create a small PR to modify these style changes, 

That would be one PR introducing the changes and another reverting the changes. That doesn't make sense because none of the PRs are merged yet. So we can just not introduce the changes in the first plane. You can do that by reverting the changes, putting them in a commit, and squashing them with the commit that introduced the changes. If the commit is far behind, you can interactively rebase the history to do it. If you're not used to doing that, just take your time because that's something we need to do all the time so it's a skill we need.

Reply:
> > It's true it's annoying. But style changes are fine.
> 
> I agree with annoying more than fine. lol

I meant that it's not ideal to mix style changes with functional changes in the same PR—it diverts focus from the main purpose. The current code does have trailing spaces, which isn’t great either. That’s why I proposed addressing it in a separate PR to keep things clean and focused.

> You can do that by reverting the changes, putting them in a commit, and squashing ...

 I realize there might have been some confusion about my approach. Here’s what I was thinking:
 
1.  #651 for Style Changes: I've already created #651, which exclusively contains the style changes that are also currently in #648.

2. Merge #651: The plan is to merge #651 first, which includes all the style changes.

3. Rebase #648: After #651 is merged, I'll rebase #648. This will effectively remove the style changes from #648’s diff, as those changes will now be part of the main branch.

This strategy ensures that #648 focuses strictly on functionality without the noise of style changes. Does this sound like a good approach to you?

Reply:
The only problem it caused was some annoyance when reviewing the PR. Apart from that, they're just trailing spaces. 

We're discussing this like it's rocket science. We don't need to fix anything; leaving them as they are is okay. The project can move on with or without these tailing spaces. 

You can remove them from this PR by squashing the changes or leaving them as they are here, as the PR has already been reviewed this way. Whatever you decide, we can keep it the new way.